torchgeo/datamodules/skippd.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SKy Images and Photovoltaic Power Dataset (SKIPP'D) datamodule."""
+
+from typing import Any
+
+from ..datasets import SKIPPD
+from .geo import NonGeoDataModule
+from .utils import dataset_split
+
+
+class SKIPPDDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the SKIPP'D dataset.
+
+    Implements 80/20 train/val splits on train_val set.
+    See :func:`setup` for more details.
+    """
+
+    def __init__(
+        self,
+        batch_size: int = 64,
+        val_split_pct: float = 0.2,
+        num_workers: int = 0,
```

Comment:
Would prob be better to have num_workers first to match all other datamodules

Reply:
okay, the order is a bit mixed in the other datamodules, but can fix that in another PR